lib/config/fileconf_test.go

```diff
@@ -476,46 +477,36 @@ func TestX11Config(t *testing.T) {
 					"enabled": "yes",
 				}
 			},
-			expectError: require.NoError,
 			expectX11Config: &x11.ServerConfig{
 				Enabled:       true,
 				DisplayOffset: x11.DefaultDisplayOffset,
-				MaxDisplay:    x11.DefaultMaxDisplay,
+				MaxDisplay:    x11.DefaultDisplayOffset + x11.DefaultMaxDisplays,
 			},
 		}, {
-			desc: "enabled value invalid",
+			desc: "x11 enabled value invalid",
 			mutate: func(cfg cfgMap) {
 				cfg["ssh_service"].(cfgMap)["x11"] = cfgMap{
-					"enabled":        "no",
-					"display_offset": "100",
+					"enabled": "maybe",
 				}
 			},
-			expectError: require.Error,
-		}, {
+			expectConfigError: func(t require.TestingT, err error, i ...interface{}) {
+				require.Error(t, err)
+				require.True(t, true, trace.IsBadParameter(err))
```

Comment:
Note that the require.True invocation is wrong, `trace.IsBadParameter(err)` is used as a description.

```suggestion
				require.True(t, trace.IsBadParameter(err), "got err = %v, want BadParameter", err)
```

Same for others.